### ISSUE TITLE
Selected Area is not applying to all Selected Area only the first Tile

### ISSUE THREAD

**Issue by alexgoldblat** [ricardoquesada/vchar64#34]
On Tiles pad , When i Select more than 1 Tile lets say 4x4 Chars , and make CTRL+SHIFT+C its only apply to one tile instead of all selected tiles. for me its a bit awkward :-1:  it should apply to all selected tiles. this is how is should work my friend !

**Comment by ricardoquesada** [comment]
yes. in order to be consistent, all the tiles transform functions should be applied to all the selected tiles/chars. so, it is not a trivial thing to do.
